.phpcs.xml

```diff
@@ -15,6 +15,9 @@
     <rule ref="Generic.Commenting.DocComment.MissingShort">
         <severity>0</severity>
     </rule>
+    <rule ref="Generic.Files.LineLength.TooLong">
```

Comment:
I am adding this because now that we have flame graph assertions is easy to exceed 120 chars, mostly with long resource names.
I tried a few different approaches, e.g. multi-line strings or concatenating blocks with `"..." . \n "..."` but neither one seemed to be more readable to me. I made this call, totally open to revisit this.